Add docs (#318)
mt-mods/technic: technic/doc/tools.md

This electric tool is a substantial upgrade above the pickaxe that you start mining with. The laser works by drilling directly forward 7 blocks making a path that's roughly 1-block in diameter. The laser is electric, requiring charging in a battery box (any voltage). As soon as an LV electrical system is set up, this is a great next step.

## Multimeter

<img src="../textures/technic_multimeter.png"/>

> This electric tool is used for quick lookup at an electric network. It works by clicking on an electric node (cable, machine, generator, switchting station), and gives values about : the position of the attached switching station, its current status, the lag, the batteries, the supply and the demand, and the total of each type of node.

Reviewer: For multimeter it would be very good to actually document values it displays, few are obvious but many are not if player isn't very familiar with technical stuff under the hood.
Maybe table with descriptions for each value displayed by multimeter.

This electric tool is a substantial upgrade above the pickaxe that you start mining with. The laser works by drilling directly forward 7 blocks making a path that's roughly 1-block in diameter. The laser is electric, requiring charging in a battery box (any voltage). As soon as an LV electrical system is set up, this is a great next step.

## Multimeter

<img src="../textures/technic_multimeter.png"/>

This electric tool is used for quick lookup at an electric network. It works by clicking on an electric node (cable, machine, generator, switchting station), and gives values about : the position of the attached switching station, its current status, the lag, the batteries, the supply and the demand, and the total of each type of node.

| Property         | Example Value | Unit       | Meaning                                                       |
|------------------|---------------|------------|---------------------------------------------------------------|

> | Ref. point       | 61, 19, 62    | Coordinate | The location of the switching station attached to the network |

Reviewer: This actually isn't pointing to switching station but 1 node below it which is cable.
While it is very often directly below switching station this is not always the case, there does not have to be switching station above it.

Actually this information is technical detail that isn't too simple when you dive into it.
Reason it is included is that I first created multimeter to aid debugging during development but just tried to make it also usable by regular players.

Reviewer: Not sure what to change in description though... Network ID is derived from this coordinate (for both actual network implementation and for ID that shows up in multimeter).
Coordinate is also used with administrative commands, same location those commands will show.

> | Ref. point       | 61, 19, 62    | Coordinate | The location of the switching station attached to the network |

Reviewer: Here's suggestion for technically correct description, probably can't be made much simpler
```suggestion
| Ref. point       | 61, 19, 62    | Coordinate | The location of the cable below the switching station that initially triggered initialization of this network |
```

| Activated        | yes           | active     | The current state of the network                              |

> | Timeout          | 1800.0        | s          | The timeout of the network                                    |

Reviewer: Maybe
```suggestion
| Timeout          | 1800.0        | s          | Time remaining, in seconds, before the network will shut down |
```

| Lag              | 0.60          | ms         | The lag generated by the network                              |

> | Skip             | 0             | cycles     | How many cycles are skipped                                   |

Reviewer: To make in clear what affects this maybe a little addition:
```suggestion
| Skip             | 0             | cycles     | How many cycles are skipped when network lag value is high    |
```

> | Supply           | 6000          | EU         | How many power is supplied by generators                      |
> | Demand           | 4000          | EU         | How many power is required by consumers                       |

Reviewer: ```suggestion
| Supply           | 6000          | EU         | How much power is supplied by generators                      |
| Demand           | 4000          | EU         | How much power is required by consumers                       |
```

| Battery charge   | 2911956       | EU         | The total of energy stored in batteries                       |
| Battery charge   | 97.07         | %          | The energy stored per the maximum that can be stored          |
| Battery capacity | 3000000       | EU         | The maximum amount of energy that can be stored in batteries  |

> | Nodes            | 47            | count      | The number of nodes of the network                            |

Reviewer: ```suggestion
| Nodes            | 47            | count      | The number of nodes of the network, excluding switch stations |
```

Reviewer: For multimeter, would probably be good to have switching station count as a separate measurement and also make waypoint functionality display waypoints for switching stations.
Not for this pull request but good update now there's actually easy straightforward way to get/store positions of all connected switching stations.
